pages/data-fetcher.tsx

```diff
@@ -137,15 +169,32 @@ const GetData: NextPage = () => {
                 <p className="help is-danger">{addressError}</p>
               )}
               {(interfaces.isErc725X || interfaces.isErc725Y) && (
-                <p className="help is-success">
+                <p className="help is-success mt-3">
                   ERC725X: {interfaces.isErc725X ? '✅' : '❌'} - ERC725Y{' '}
                   {interfaces.isErc725Y ? '✅' : '❌'}
                 </p>
               )}
             </div>
 
             <div className="field">
-              <label className="label">Data key</label>
+              <label className="label">Data Key</label>
+
+              <div className="select mb-4 dataKeyInput">
+                <select
+                  onChange={(e) => onDataKeyChange(e.target.value)}
+                  className="dataKeyInput"
```

Comment:
I think the name of that class is not very good as the only thing it does it to set the width to 100%

So maybe something like "fullwidth" would make more sense.

I'm curious why the default bulma select implementation dies not work tho

https://bulma.io/documentation/form/select/

I'm on mobile and can't check